plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/manager/VmwareStorageManagerImpl.java

```diff
@@ -1306,12 +1307,12 @@ private void setVolumeToPathAndSize(List<VolumeObjectTO> volumeTOs, Map<String,
             // get volume's chain size for this VM snapshot; exclude current volume vdisk
             DataStoreTO store = volumeTO.getDataStore();
             ManagedObjectReference morDs = getDatastoreAsManagedObjectReference(baseName, hyperHost, store);
-            long size = getVMSnapshotChainSize(context, hyperHost, baseName + ".vmdk", morDs, newPath);
-            size = getVMSnapshotChainSize(context, hyperHost, baseName + "-*.vmdk", morDs, newPath);
+            long size = getVMSnapshotChainSize(context, hyperHost, baseName + ".vmdk", morDs, newPath, vmName);
```

Comment:
shouldn't this call be removed @sureshanaparti?

Reply:
yes @Pearl1594, already created a separate PR (https://github.com/apache/cloudstack/pull/5720).